torch/_dynamo/variables/builtin.py

```diff
@@ -1512,3 +1512,17 @@ def call_not_(self, tx, a):
     call_ne = _comparison
     call_is_ = _comparison
     call_is_not = _comparison
+
+    def call_all(self, tx, it):
+        if isinstance(it, ListIteratorVariable):
+            try:
+                # Constant fold
+                return ConstantVariable.create(all(it.as_python_constant()))
```

Comment:
I think this needs to be written a bit more tightly, right now you're just sort of assuming you can always `proxy_args_kwargs` all the arguments which won't always be true.

In fact, I would probably suggest writing a polyfill for all and inlining into it. I'll push a commit to this PR showing how to do it.

Reply:
before i get to it

"[12:00](https://pytorch.slack.com/archives/C05SDC762BT/p1695657635610349)
call inline_function whatever on a pure python implementation of all

[12:00](https://pytorch.slack.com/archives/C05SDC762BT/p1695657652523069)
you need to put it in some separate module and make sure it's inlinable (since all of torch is not inlinable"

Reply:
Do you mean put it in some polyfill torch module, then `call_module` on it?